db/migration/src/m20250219_000020_logs_add_index.rs

```diff
@@ -0,0 +1,47 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_index(
+                sea_query::Index::create()
+                    .if_not_exists()
+                    .name("idx_log_status_block_number_processed")
+                    .table(LogStatus::Table)
+                    .col((LogStatus::BlockNumber, IndexOrder::Asc))
+                    .col(LogStatus::Processed)
+                    .to_owned(),
+            )
+            .await
+    }
+
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .drop_index(Index::drop().name("idx_log_status_block_number_processed").to_owned())
+            .await
+    }
+}
+
+#[derive(DeriveIden)]
+enum LogStatus {
+    Table,
+    // Values to identify the log.
+    BlockNumber,
+    #[allow(dead_code)]
+    TransactionIndex,
```

Comment:
You dont have to have all the enum members here, it will work also if  those unused are omitted from the enum.

Reply:
ok good to know